### DISCUSSION TITLE
Status codes by name?

### DISCUSSION THREAD

**Discussion by AustinScola** [aio-libs/aiohttp#5790] [Q&A] [answered]
Is there a way to access status codes by name rather than using integers?

I see that `aiohttp.web` has [server exceptions](https://docs.aiohttp.org/en/stable/web_quickstart.html#exceptions) that are being deprecated, but that is the closest that I can find? The documentation generally uses the integer values like `200` so I'm guessing this isn't currently available? I'm imagining something like `from aiohttp.statuses import Ok, NotFound, ...`.

P.S. If this is something that is determined to be worth adding I would gladly contribute!

**Comment by webknjaz** [comment, marked as answer]
I don't think it is necessary to reimplement this in aiohttp. One could easily just use the constants provided by stdlib: https://docs.python.org/3/library/http.html#http-status-codes.

**Reply by AustinScola** [reply]
Great, I didn't know about that package. Thanks for pointing it out to me!

**Comment by Dreamsorcerer** [comment]
> I see that `aiohttp.web` has [server exceptions](https://docs.aiohttp.org/en/stable/web_quickstart.html#exceptions) that are being deprecated

As a side point, I don't think they are deprecated at all. _returning_ them is deprecated.

**Reply by AustinScola** [reply]
Thanks for the correction. So they can still be raised just not returned?

**Reply by Dreamsorcerer** [reply]
Correct, only raising will be supported in aiohttp 4+.
